docs/source/models.rst

```diff
@@ -115,6 +115,11 @@ Probabilitic Matrix Factorization (PMF)
 .. automodule:: cornac.models.pmf.recom_pmf
    :members:
 
+Non-negative Matrix Factorization (NMF)
```

Comment:
We need to organize this according to the order of models in the main README file